pytorch/training/docker/1.4.0/py2/Dockerfile.gpu

```diff
@@ -51,7 +51,6 @@ RUN apt-get update \
     wget \
     zlib1g-dev \
  && apt-get remove -y cuda-cufft-dev-10-1 \
-    cuda-curand-dev-10-1 \
```

Comment:
just want to double check -- is this change not necessary in inference containers?

Reply:
We need it for training images only.